lua/lvim/core/nvimtree.lua

```diff
@@ -170,9 +170,9 @@ end
 function M.start_telescope(telescope_mode)
   local node = require("nvim-tree.lib").get_node_at_cursor()
   local abspath = node.link_to or node.absolute_path
-  local is_folder = node.has_children and true
+  local is_folder = node.open ~= nil
   local basedir = is_folder and abspath or vim.fn.fnamemodify(abspath, ":h")
-  vim.api.nvim_command("Telescope " .. telescope_mode .. " cwd=" .. basedir)
+  vim.api.nvim_command("Telescope " .. telescope_mode .. " cwd=" .. string.gsub(basedir, " ", "\\ "))
```

Comment:
```suggestion
  require("telescope.builtin")[telescope_mode] {
    cwd = vim.fn.fnameescape(basedir),
  }
```

Reply:
There is no need to escape basedir since this is passed as an argument directly to telescope, and being an absolute path given by nvimtree it's already ready to use

Reply:
So it probably was just a problem translating vimL -> lua.